utils/inputValidation/flagSets.js

```diff
@@ -0,0 +1,28 @@
+const errorWrapper = require('./wrapper/error');
+const okWrapper = require('./wrapper/ok');
+const lang = require('../lang');
+const validateFlagSet = require('./flagSet');
+const { EMPTY_FLAG_SETS, NULL_FLAG_SETS } = require('../constants');
+
+const validateFlagSets = (maybeFlagSets) => {
+  // eslint-disable-next-line eqeqeq
+  if (maybeFlagSets == undefined) return errorWrapper(NULL_FLAG_SETS);
+
+  maybeFlagSets = maybeFlagSets.split(',');
+
+  if (maybeFlagSets.length > 0) {
+    let validatedArray = [];
```

Comment:
why array and not dictionary (js obj) then return all the keys?

Reply:
To be consistent with how we are validating splits parameters